_posts/2016-02-04-ombulabs-products.markdown

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: "Announcing The Ombu Labs Shop!"
+date: 2016-03-17 18:35:00
```

Comment:
@cecilia-ombulabs could you update the date on this post? You can use today's date, so that it shows up first in the index, otherwise it goes down a bit because of more recent posts.